ui/src/components/Comments/index.tsx

```diff
@@ -53,10 +65,21 @@ class Comment extends React.PureComponent<ICommentProps> {
                             <span className='handle'>{this.props.comment.attributedTo.account}</span>
                         </div>
                     </a>
-                    <span aria-hidden="true">·</span>
                     <a href={this.props.comment.url} className='permalink'>
                         <time>{this.props.comment.publishedAt.toLocaleString()}</time>
                     </a>
+                    <button className="context-menu" aria-label="More" onClick={() => this.onClickShowMenu()}>
+                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" width="24" height="24" stroke-width="1.5" stroke="currentColor">
+                            <path stroke-linecap="round" stroke-linejoin="round" d="M6.75 12a.75.75 0 11-1.5 0 .75.75 0 011.5 0zM12.75 12a.75.75 0 11-1.5 0 .75.75 0 011.5 0zM18.75 12a.75.75 0 11-1.5 0 .75.75 0 011.5 0z" />
```

Comment:
I get a warning in the console about `stroke-width`, `stroke-linecap`, and `stroke-linejoin`. According to PyCharm, they should be `strokeWidth`, `strokeLinecap`, and `strokeLinejoin`.

Reply:
Thank you @stevencrader , I'll replace that.
Do you know that you can suggest changes in GitHub like this?
```suggestion
                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" width="24" height="24" strokeWidth="1.5" stroke="currentColor">
                            <path strokeLinecap="round" strokeLinejoin="round" d="M6.75 12a.75.75 0 11-1.5 0 .75.75 0 011.5 0zM12.75 12a.75.75 0 11-1.5 0 .75.75 0 011.5 0zM18.75 12a.75.75 0 11-1.5 0 .75.75 0 011.5 0z" />
```

Reply:
Resolved with https://github.com/Podcastindex-org/web-ui/pull/253/commits/09396eb87e42ece3d05413e265c7af83c897e26e.

Reply:
I didn't know about that. Thanks for the tip.